src/Functions/toRelativeYearNum.cpp

```diff
@@ -11,7 +11,41 @@ using FunctionToRelativeYearNum = FunctionDateOrDateTimeToSomething<DataTypeUInt
 
 REGISTER_FUNCTION(ToRelativeYearNum)
 {
-    factory.registerFunction<FunctionToRelativeYearNum>();
+    FunctionDocumentation::Description description_toRelativeYearNum = R"(
+Converts a date or date with time to the number of years elapsed since a certain fixed point in the past.
+    )";
+    FunctionDocumentation::Syntax syntax_toRelativeYearNum = R"(
+toRelativeYearNum(date)
+    )";
+    FunctionDocumentation::Arguments arguments_toRelativeYearNum = {
+        {"date", "Date or date with time. [`Date`](../data-types/date.md)/[`DateTime`](../data-types/datetime.md)/[`DateTime64`](../data-types/datetime64.md)."}
```

Comment:
Can we please replace `/` by `or` for better readability?

Reply:
@rschu1ze keep in mind they will show up like this when rendered
![Screenshot 2025-06-17 at 12 29 06](https://github.com/user-attachments/assets/2bb02c08-b1be-42c3-9270-32d1ce03fcf9). Maybe we should consider adding some helper methods to FunctionDocumentation, something like `FunctionDocumentation::formatReturnType("Date to convert to a time.", return::DateTime, return::DateTime64)` and it will produce the formatted string: `Date to convert to a time. [`DateTime`](../data-types/datetime.md)/[`DateTime64`](../data-types/datetime64.md).`

Reply:
Hmm, fair point. I will think about it ... Today, `FunctionDocumentation` contains this:

```cpp
struct Argument
{
    std::string name;
    std::string description;
};
using Arguments = std::vector<Argument>;
```

I guess what we need is

```cpp
struct Argument
{
    std::string name;
    std::string description;
    std::vector<std::string> types;
};
using Arguments = std::vector<Argument>;
```

as well as a corresponding adjustment in `std::string FunctionDocumentation::argumentsAsString() const` to concatenate all argument types in the format we like them to be, right?